emcc.py

```diff
@@ -2011,8 +2008,8 @@ def get_clang_command_asm(src_file):
         if specified_target:
           cmd += ['-o', specified_target]
         logger.debug("running (for precompiled headers): " + cmd[0] + ' ' + ' '.join(cmd[1:]))
-        shared.print_compiler_stage(cmd)
-        return run_process(cmd, check=False).returncode
+        shared.check_call(cmd)
+        return 0
```

Comment:
Here too. The old code will return the nonzero exiti code in case of an error but the new code will terminate the program.

Reply:
Ditto, in both cases the non-zero return code is supposed be fatal and continues to be after this change.